src/traces/parcoords/lines.js

```diff
@@ -1,5 +1,17 @@
 'use strict';
 
+// Fix for Chrome 128
+var INF = 1e38;
```

Comment:
These are all evaluated as single precision floats? So I should compare it to 3.4e38, the max single-precision value? This seems reasonable then, a bit of margin in case they get added together or something.

Reply:
I was not able to increase it to 1e39 on latest Chrome; so I used 1e38.
Do you suggest we use 3.4e38 instead?

Reply:
1e38 is good. We don’t want to get too close to the 3.4e38 limit of single precision, in case some operations push our input over that limit. I just wanted confirmation that these routines only ever apply to single-precision floats